README.md

```diff
@@ -252,7 +252,7 @@ ActiveAdmin.register Page do
     actions
   end
 
-  form do |f|
+  form html: { class: "edit_atomic_content" } do |f|
```

Comment:
Thanks, I just following the pattern from all the id's above in that file. We should probably establish some real conventions and document them somewhere, cause this project doesn't seem to be adhering to many so far. 

Made the edits, thanks!